tests/test_configuration.py

```diff
@@ -9,7 +9,7 @@ def test_duplicate_config_key(self):
             tempfile.write(config)
             tempfile.flush()
             with self.assertRaises(KeyError):
-                read_variables(tempfile.name)
+                Configurator().read_variables(tempfile.name)
```

Comment:
If the intention to make read_variable a method is to test the object construction - that is actually doing nothing see above - this should not be done implicit but with a dedicated test.